index.rst

```diff
@@ -2,24 +2,209 @@
+For example, when a CSC is in the STANDBY state and receives a ``start`` command we expect it to publish a ``summaryState`` event indicating that it transitioned to the DISABLED state.
+
+Activity Sequence Diagrams are broadly used in system design, architecture and systems engineering.
+They allow us to specify _how_ a system works by showing the flow of information between components.
+As such, they provide a way to demonstrate the sequence of communication during a particular activity, which are useful to support and add context to verification artifacts.
+
+Since all communications between components are stored in the EFD, we can produce Activity Sequence Diagrams for any operation after-the-fact, just like we do with :ref:`Robot Framework <robot-framework>` reports, using a reporting tools specifically designed for this purpose.
+
+The :ref:`figure below <fig-ataos-state-transition>` is an example diagram showing the state transition of the ATAOS CSC from STANDBY to ENABLED.
+It shows the commands sent by the SAL Script to the ATAOS CSC, the acknowledgements published by the CSC after receiving the command and the subsequent events (with payload information).
+
+.. figure:: /_static/ataos-state-transition.png
+    :name: fig-ataos-state-transition
+    :target: ../_images/ataos-state-transition.png
+    :alt: ATAOS state transition Activity Diagram
+
+    Activity Sequence Diagram for the transition from STANDBY to DISABLED and subsequently to ENABLED for the ATAOS CSC.
+
+Jira Test Cases
+===============
+
+Jira test cases provide a way to organize, plan and track verification test execution.
+They can contain detailed information about test execution and provide valuable verification artifacts.
+
+On the other hand, because they are detached from the actual products, they cannot automatically capture changes in the system, which needs to be manually included/updated by a test executor.
+Furthermore, executing these test cases alongside the actual test requires dedicate personnel to manually log information, slowing down the process and introducing substantial overhead.
 
-Requirement verification is an important step in the delivery of any project and, given the nature of software development, it can become challenging. Here, we detail the various strategies and tools developed to support software requirement verification.
+We acknowledge that Jira Test Cases can be helpful in planning more complicated verification activities (e.g. those that can not be accomplished by automated testing).
+Nevertheless, for the automated test suite we recommend using a simplified test case which contains the minimum information required (e.g. the requirement being verified and an explanation of the test).
+The verification artifacts can then contain a link to the :ref:`Robot Framework <robot-framework>` report and possibly an :ref:`Activity Diagram <activity-sequence-diagrams>`, all of which can be generated automatically following the test execution.
```

Comment:
So, this document does a good job of describing our testing procedure and how it can verify requirements - the problem now is how to sync into SE's requirements verification tracking system - which they have taken years to build and they show off at every review.  We gloss over that problem here and I am sure it is the reason Austin is very hesitant on moving forward with this.
We probably should address that here.  Maybe an argument for how TSSW & SW can look into finding a way to make this work?  Otherwise I believe there will be a lot of push back. 
Although it is already stated in this document, we may want to  re-emphasize how this will help in automating the testing to drastically reduce their verification backlog - which was also a recommendation from the JSR.  This work can show them that they are working toward that goal.

Reply:
This is an excellent point! I am thinking about how to cover this here.. I may ask for you input again once I have written something down! thank you so much for the feedback!